src/animal-quest.ts

```diff
@@ -5,6 +5,21 @@ type Seconds = `${0 | 1 | 2 | 3 | 4 | 5}${ZeroToNine}s`;
 type Minutes = `${0 | 1 | 2 | 3 | 4 | 5}${ZeroToNine}m`;
 type Hours = `${`${0 | 1}${ZeroToNine}` | `2${0 | 1 | 2 | 3}`}h`;
 
+const hostNames = {
+  maya: "Maya Higa",
+} as const;
+
+type Host = { name: HostName; link?: string };
+type HostKey = keyof typeof hostNames;
+type HostName = (typeof hostNames)[HostKey];
+
+export const hosts: Record<HostKey, Host> = {
+  maya: {
+    name: "Maya Higa",
+    link: "https://www.alveussanctuary.org/about/maya",
+  },
+};
```

Comment:
```suggestion
type Host = { name: string; link?: string };
type HostKey = keyof typeof hosts;

export const hosts = {
  maya: {
    name: "Maya Higa",
    link: "https://www.alveussanctuary.org/about/maya",
  },
} as const satisfies Record<string, Host>;
```
You've got a lot of duplicate data here, which we can simplify